### PR TITLE
feat : added product listing page

### PR DESCRIPTION
**Pull request by vivekgudipudi**
added product listing page and improved UX for mobile view

image
image
image
image
image

### CODE REVIEW

File: components/product-listing/product.html [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+                    <div class="card-ecom-price">
+                        <span class="card-ecom-price-current">Rs.98700</span>
+                        <span class="card-ecom-price-old">Rs.139900</span>
+                    </div>
+                </div>
+                <div class="card-footer">
+                    <div class="card-btn-box">
+                        <button class="btn default-primary">ADD TO CART</button>
+                    </div>
+                </div>
```
Comment (srejitk) [inline components/product-listing/product.html:290]: Would suggest using semantic elements instead of div

**Review by srejitk — APPROVED** [review]
Code looks good, have added few minor suggestions.

File: components/product-listing/product.html (hunk 1 above)
Comment (vivekgudipudi) [reply components/product-listing/product.html:290]: they are from component library, will modify them once I complete e commerce app, thanks for suggestion!

**Review by krishanu2911 — APPROVED** [review]
No changes are required Everything is good to go 
Nice use of classes in HTML👍🏻

**Pull request merged**